### ISSUE TITLE
Add "jurisdiction" field to report type

### ISSUE THREAD

**Issue by fbennett** [citation-style-language/zotero-bits#46]
Reports issued by government bodies are formatted differently from reports by private entities in some styles. See OSCOLA examples at the bottom of section 3.4.2, and the Whittaker example in section 3.4.1:

  http://www.law.ox.ac.uk/published/OSCOLA_4th_edn.pdf

Adding the "jurisdiction" field to reports would cover this case nicely.

**Comment by JohnLukeBentley** [comment]
I'd also suggest adding a "jurisdiction" field to the "statute", "patent" and "case" (?) types.

From CSL 1.0.1 http://docs.citationstyles.org/en/stable/specification.html#standard-variables
> jurisdiction
>    geographic scope of relevance (e.g. “US” for a US patent)

**Comment by fbennett** [comment]
For anyone who drives by this thread, the "MLZ extension" referred to above is now [Juris-M](https://juris-m.github.io), and the identifiers used by the system are currently hosted on GitHub as a [Legal Resource Registry](https://fbennett.github.io/legal-resource-registry). There will be some action in that space over the coming months.

**Comment by JohnLukeBentley** [comment]
@fbennett, we'll lo and behold, it looks like you've gone ahead and created everything Zotero does with robust support for legal referencing. 

I see, for a start, the "jurisdiction" field added to the "statute", "patent" and "case" types. Moreover, the Jurisdiction field is a drop down box that sources values from your "Legal Resource Registry".

Looks like you've done some enormous work there.

I've been on the cusp of designing my own legal style, because existing styles seemed not quite right, it looks like Juris-M and possibly one of your existing styles might sort me out.

Not wanting to drag this thread off topic ... On your home page <https://juris-m.github.io/> you mention "Juris-M is free software". Do you mean this in the "freedom" sense (if so I can't see where you are hosting your source code)? 

Also might github issues be a better place for discussions on Juris-M than mailing lists?

**Comment by gracile-fr** [comment]
(@JohnLukeBentley, the source code is here : https://github.com/Juris-M/zotero/tree/jurism-5.0)

**Comment by fbennett** [comment]
@JohnLukeBentley Yep. As @gracile-fr says, the source code of Juris-M is online in the clear at GitHub. It's a fork of Zotero, and so bound by the same open source license terms. I'll make a note to add a direct link to the source code repository to the website pages. The "Support" tab on the site has links to project mailing lists, feel free to pick one and subscribe for project news. The "jurism-support" list would be a good channel for floating development ideas etc.

**Comment by JohnLukeBentley** [comment]
Thanks @gracile-fr and @fbennett I'll see you on one of your lists.

**Comment by dstillman** [comment]
Would this be valuable in stock Zotero, without extended jurisdiction support? If so, thoughts on appropriate types?

**Comment by bwiernik** [comment]
There has been some discussion by @georgd about jurisdictions lately. I think it would make sense for the legal types--legal_case, bill, legislation, treat, legislation, regulation, report. It could be a standard field in stock Zotero, or it could have the controlled list used by Jurism

**Comment by adam3smith** [comment]
But would a standard field really give us much. As per Frank's OP, we'd need to somehow check content, no?

**Comment by bwiernik** [comment]
That's the thing @georgd and I were discussing a while back. Even basic legal citation for European contexts needs multi-jurisdiction support. The proposal was to permit content testing matching like is done with the `language` field, probably with a controlled vocabulary.

**Comment by fbennett** [comment]
If Zotero decides to take any steps in this direction, please contact @mlissner at the Free Law Project. They are a well established shop and have been working on these issues for years in the US context. They have code and expertise that would be useful.
